### PR TITLE
[Rule Tuning] Updated setup guide

### PR DESCRIPTION
**Pull request by Aegrah**
## Summary
Added setup guide instructions to this rule, to allow customers to more easily set up the necessary auditd rules.

### CODE REVIEW

**Comment by protectionsmachine** [comment]
## Rule: Tuning - Guidelines

These guidelines serve as a reminder set of considerations when tuning an existing rule.

### Documentation and Context

- [ ] Detailed description of the suggested changes.
- [ ] Provide example JSON data or screenshots.
- [ ] Provide evidence of reducing benign events mistakenly identified as threats (False Positives).
- [ ] Provide evidence of enhancing detection of true threats that were previously missed (False Negatives).
- [ ] Provide evidence of optimizing resource consumption and execution time of detection rules (Performance).
- [ ] Provide evidence of specific environment factors influencing customized rule tuning (Contextual Tuning).
- [ ] Provide evidence of improvements made by modifying sensitivity by changing alert triggering thresholds (Threshold Adjustments).
- [ ] Provide evidence of refining rules to better detect deviations from typical behavior (Behavioral Tuning).
- [ ] Provide evidence of improvements of adjusting rules based on time-based patterns (Temporal Tuning).
- [ ] Provide reasoning of adjusting priority or severity levels of alerts (Severity Tuning).
- [ ] Provide evidence of improving quality integrity of our data used by detection rules (Data Quality).
- [ ] Ensure the tuning includes necessary updates to the release documentation and versioning.

### Rule Metadata Checks

- [ ] `updated_date` matches the date of tuning PR merged.
- [ ] `min_stack_version` should support the widest stack versions.
- [ ] `name` and `description` should be descriptive and not include typos.
- [ ] `query` should be inclusive, not overly exclusive. Review to ensure the original intent of the rule is maintained.

### Testing and Validation

- [ ] Validate that the tuned rule's performance is satisfactory and does not negatively impact the stack.
- [ ] Ensure that the tuned rule has a low false positive rate.

**Comment by approksiu** [comment]
@Aegrah could you please follow the setup guide template mentioned in this issue: https://github.com/elastic/ia-trade-team/issues/410

**Comment by Aegrah** [comment]
@approksiu copy pasted it from the template now.

**Review by imays11 — APPROVED** [review]

File: rules/linux/persistence_user_or_group_creation_or_modification.toml [hunk 1]
```diff
@@ -17,17 +17,40 @@ license = "Elastic License v2"
 name = "User or Group Creation/Modification"
 risk_score = 21
 rule_id = "fcf733d5-7801-4eb0-92ac-8ffacf3658f2"
-setup = """## Setup
-This rule requires the use of the `auditd_manager` integration. `Auditd_manager` is a tool designed to simplify and enhance the management of the audit subsystem in Linux systems. It provides a user-friendly interface and automation capabilities for configuring and monitoring system auditing through the auditd daemon. With `auditd_manager`, administrators can easily define audit rules, track system events, and generate comprehensive audit reports, improving overall security and compliance in the system. The following steps should be executed in order to install and deploy `auditd_manager` on a Linux system.
+setup = """### Auditd Manager Integration Setup
+The Auditd Manager Integration receives audit events from the Linux Audit Framework which is a part of the Linux kernel.
```
Comment (shashank-elastic) [inline rules/linux/persistence_user_or_group_creation_or_modification.toml:21]: The format seems a bit off from the templates we have [here](https://docs.google.com/document/d/1J4ClwoAkhcP4BDTw94Bwb3nC_6kXsl5pu99Um8Ddt58/edit#heading=h.akgoryidjhd5)

Its good to keep it consistent across the rules, if we dont have a specific reason as to why this has changed here.

Comment (Aegrah) [reply rules/linux/persistence_user_or_group_creation_or_modification.toml:21]: I copy pasted it from that doc..

File: rules/linux/persistence_user_or_group_creation_or_modification.toml [hunk 2]
```diff
@@ -17,17 +17,40 @@ license = "Elastic License v2"
 name = "User or Group Creation/Modification"
 risk_score = 21
 rule_id = "fcf733d5-7801-4eb0-92ac-8ffacf3658f2"
-setup = """## Setup
-This rule requires the use of the `auditd_manager` integration. `Auditd_manager` is a tool designed to simplify and enhance the management of the audit subsystem in Linux systems. It provides a user-friendly interface and automation capabilities for configuring and monitoring system auditing through the auditd daemon. With `auditd_manager`, administrators can easily define audit rules, track system events, and generate comprehensive audit reports, improving overall security and compliance in the system. The following steps should be executed in order to install and deploy `auditd_manager` on a Linux system.
+setup = """### Auditd Manager Integration Setup
```
Comment (shashank-elastic) [inline rules/linux/persistence_user_or_group_creation_or_modification.toml:20]:
```suggestion
setup = """## Setup

This rule requires data coming in from Auditd Manager .

### Auditd Manager Integration Setup
```

File: rules/linux/persistence_user_or_group_creation_or_modification.toml (hunk 1 above)
Comment (shashank-elastic) [reply rules/linux/persistence_user_or_group_creation_or_modification.toml:21]: Have added the suggestion on what was missing

**Review by shashank-elastic — APPROVED** [review]
Approving : Post committing the Setup Suggestions

File: rules/linux/persistence_user_or_group_creation_or_modification.toml [hunk 3]
```diff
@@ -18,16 +18,43 @@ name = "User or Group Creation/Modification"
 risk_score = 21
 rule_id = "fcf733d5-7801-4eb0-92ac-8ffacf3658f2"
 setup = """## Setup
-This rule requires the use of the `auditd_manager` integration. `Auditd_manager` is a tool designed to simplify and enhance the management of the audit subsystem in Linux systems. It provides a user-friendly interface and automation capabilities for configuring and monitoring system auditing through the auditd daemon. With `auditd_manager`, administrators can easily define audit rules, track system events, and generate comprehensive audit reports, improving overall security and compliance in the system. The following steps should be executed in order to install and deploy `auditd_manager` on a Linux system.
+
+This rule requires data coming in from Auditd Manager .
```
Comment (Aegrah) [inline rules/linux/persistence_user_or_group_creation_or_modification.toml:22]:
```suggestion
This rule requires data coming in from Auditd Manager.
```

File: rules/linux/persistence_user_or_group_creation_or_modification.toml (hunk 1 above)
Comment (shashank-elastic) [reply rules/linux/persistence_user_or_group_creation_or_modification.toml:25]: Discussed on slack. This was a more of a main section introduction missing!

**Pull request merged**